lib/rls/api.rb

```diff
@@ -59,6 +60,25 @@ def players(*request_data)
       response.map { |e| Player.new(e) }
     end
 
+    # Searches RLS's database for players matching a given display name.
+    # The response is paginated.
+    #
```

Comment:
Could you also add an example of search here, maybe just copy the psuedocode from our DM and touch it up a little if necessary?